x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportGetDeploymentStatsAction.java

```diff
@@ -302,6 +302,7 @@ protected void taskOperation(
                     // avoid reporting the average time as 0 if count < 1
                     (stats.get().getTimingStats().getCount() > 0) ? stats.get().getTimingStats().getAverage() : null,
                     stats.get().getPendingCount(),
+                    stats.get().getErrorCount() > 0 ? stats.get().getErrorCount() : null,
```

Comment:
If there are no errors setting `null` here means the field will not be in the response